chapters/statements.tex

```diff
@@ -299,6 +303,9 @@ \subsection{While-Statement}\label{while-statement}
 \item
   If the \lstinline!expression! of the \lstinline!while!-statement is true, the entire body of the \lstinline!while!-statement is executed (except if a \lstinline!break!-statement, see \cref{break-statement}, or a \lstinline!return!-statement, see \cref{return-statements}, is executed), and then execution proceeds at step~\ref{while-execution-top}.
 \end{enumerate}
+Neither the \lstinline!expression! nor any expression in the body shall generate events.
+In order to ensure that there are no events it is recommended to use explicit \lstinline!noEvent! in \lstinline!while!-statements in models.
```

Comment:
This sentence is just strange, and it following statement about deprecated behavior is sufficient:
```suggestion
```